torch/nn/modules/rnn.py

```diff
@@ -373,50 +373,50 @@ class RNN(RNNBase):
         bias: If ``False``, then the layer does not use bias weights `b_ih` and `b_hh`.
             Default: ``True``
         batch_first: If ``True``, then the input and output tensors are provided
-            as `(batch, seq, feature)`. Default: ``False``
+            with the batch dimension before the sequence dimension. Note that this does
```

Comment:
nit: "as (batch, seq, feature) instead of (seq, batch, feature)" makes it more concise and doesn't require the reader to go to the Inputs/Outputs section